src/unix/notbsd/linux/mod.rs

```diff
@@ -248,6 +248,11 @@ pub const ABMON_10: ::nl_item = 0x20017;
 pub const ABMON_11: ::nl_item = 0x20018;
 pub const ABMON_12: ::nl_item = 0x20019;
 
+// This is defined here instead of in ../mod.rs because Android doesn't expose
+// it yet; the change is on master, but NDK 12 beta doesn't include it, so it
+// will likely be absent for a while.
```

Comment:
It's ok to omit comments like these, we tend to not leave comments generally in libc (as it's just bindings)